.github/labeler.yml

```diff
@@ -16,3 +16,9 @@
 
 "changelog-entry-required":
   - all: ["!doc/nrf/releases/release-notes-changelog.rst"]
+
+"ble mesh":
+  - "subsys/bluetooth/mesh/*"
+  - "include/bluetooth/mesh/*"
+  - "samples/bluetooth/mesh/*"
+  - "doc/nrf/libraries/bluetooth_services/mesh/*"
```

Comment:
Mesh user guides are located at `doc/nrf/` and file names start with `ug_bt_mesh_`.